### DISCUSSION TITLE
Ctrl+Tab hotkey has no appearance in the settings and has no sound feedback

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#21708] [Q&A] [answered]
https://user-images.githubusercontent.com/86934170/208405272-bc2750b6-1540-4780-909e-c75faf8fe931.mp4

**Comment by bdach** [comment]
This should not be added to settings as ctrl-tab is a pretty standard hotkey for tab navigation, so it should not be user-editable.

**Reply by Loreos7** [reply]
Should I know about it instead? Like I figured it out accidentally, didn't know it has been existed since begging. I presume you can add a line in keybindings without an ability to change it but just to know (using grey color, for example) it's here in the game.

**Comment by peppy** [comment]
@nekodex one to add to your list if it's not on there already.

**Reply by nekodex** [reply]
Switching between tabs via keyboard is on the list already - https://github.com/ppy/osu/issues/14082#issuecomment-895127787

**Reply by Loreos7** [reply]
Alright then just pay attention on appearance. I somehow should now about every keybinding existing in the game (except developer ones). It may have look like others but be gray and unselectable.
![image](https://user-images.githubusercontent.com/86934170/211783715-0b7f0769-c76e-4c6f-a1b8-d15f293d6026.png)

**Comment by Loreos7** [comment, marked as answer]
Fixed in SongSelectV2. Don't think Ctrl+TAB hotkey has to be customisable.
